src/normalizer/mod.rs

```diff
@@ -84,13 +75,13 @@ impl NormalizerOption {
 /// Trait defining a normalizer.
 pub trait Normalizer: Sync + Send {
     /// Normalize the provided [`Token`].
+    /// Options can be set using the provided [`NormalizerOption`].
     ///
     /// A Normalizer can return several `Token`s.
-    fn normalize<'o>(&self, token: Token<'o>) -> Box<dyn Iterator<Item = Token<'o>> + 'o>;
+    fn normalize<'o>(&self, token: Token<'o>) -> Box<dyn Iterator<Item = Token<'o>> + 'o> {
+        self.normalize_with_option(token, NormalizerOption::default())
+    }
```

Comment:
We may remove the `normalize_with_option` method and only keep `normalize` adding a new parameter.
Moreover, I think you can pass the `NormalizerOption` by reference avoiding some clone, but because it only contains a boolean, I'm not sure it's necessary here 🤔 
```suggestion
    fn normalize<'o>(&self, token: Token<'o>, options: NormalizerOption) -> Box<dyn Iterator<Item = Token<'o>> + 'o>;
```

Reply:
Passing `NormalizerOption` by reference caused some issues. For example:
```rust
impl<'o> Tokenize<'o, Vec<u8>> for &'o str {
    fn tokenize(&self) -> ClassifiedTokenIter<'o, '_, Vec<u8>> {
        self.segment().normalize(NormalizerOption::default()).classify()
    }

   ...
}
```
if `normalize()` expects a `&NormalizerOption`, then we can't create a `NormalizerOption` value inside of `tokenize()` and return a reference to it because the reference would be invalid as soon as the call to `tokenize()` ends. We could pass a `&NormalizerOption` to `tokenize()` and simply pass it along to `normalize()` but I think that would break the current API.

For now, I simply derived the `Copy` trait for `NormalizerOption` and then removed all calls to `clone()`. Let me know if that suffices.

Reply:
While NormalizerOption is a small structure, I'm totally ok with it :+1: